<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="play_sound">
        <property name="text">
         <string>播放声音</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="input_binary"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="start_record">
        <property name="text">
         <string>开始录音</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_record">
        <property name="text">
         <string>停止录音 </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>331</width>
     <height>118</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="demodulation">
      <property name="text">
       <string>解调</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="decode_binary"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>411</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="reload">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="draw">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>绘制图表</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
